app/common/global/event-handlers.js

```diff
@@ -1,16 +1,25 @@
 module.exports = [
     '$rootScope',
     '$location',
+    '$window',
+    'Util',
 function (
     $rootScope,
-    $location
+    $location,
+    $window,
+    Util
 ) {
     // Setup PL layout and switching function
     $rootScope.globalLayout = 'layout-a';
     $rootScope.setLayout = function (layout) {
         $rootScope.globalLayout = layout;
     };
-
+    // Set embed mode
+    $rootScope.embedUrl = Util.deploymentUrl();
+    $rootScope.globalEmbed = ($window.self !== $window.top) ? true : false;
```

Comment:
I really dislike that we're adding more stuff to `$rootScope` .. 
rootScope is inherited by all child scopes, and then dirty checked in each digest cycle. ie. more things on rootScope, slower digest. It'd make sense for stuff like site name, if we inherited that everywhere. But we're mostly only checking `globalEmbed` within directives where we have to inject rootScope anyway.

We can keep this for now because I'm not sure what the right solution is.. but if you're got a better idea we should tend towards that over time.